test/test_transforms.py

```diff
@@ -2266,5 +2267,35 @@ def test_random_grayscale_with_grayscale_input():
     torch.testing.assert_close(F.pil_to_tensor(output_pil), image_tensor)
 
 
+# TODO: remove in 0.17 when we can delete functional_pil.py and functional_tensor.py
+@pytest.mark.parametrize(
+    "import_statement",
+    (
+        "from torchvision.transforms import functional_pil",
+        "from torchvision.transforms import functional_tensor",
+        "from torchvision.transforms.functional_tensor import resize",
+        "from torchvision.transforms.functional_pil import resize",
+    ),
+)
+@pytest.mark.parametrize("from_private", (True, False))
+def test_functional_deprecation_warning(import_statement, from_private):
```

Comment:
I realize this is slightly overkill, we don't need `assert_run_python_script` to check the warnings of the deprecated (public) files. But we kinda still need it to properly check the *lack* of warnings for the now private files, so I mixed both. The whole thing will be removed soon anyway.